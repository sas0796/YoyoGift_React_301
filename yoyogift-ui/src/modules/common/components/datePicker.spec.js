import React from "react";
import { mount } from "enzyme";

import DatePickers from "./datePicker";

describe("Date Picker Test Suite", () => {
  it("should render expected HTML", () => {
    let classes = {
      container: {marginTop: "1px"},
      label: {marginTop: "1px"},
      defaultValue: {marginTop: "1px"},
      textField: {marginTop: "1px"}
    };
    expect(mount(<DatePickers classes={classes} />).html()).toMatchSnapshot();
  })
})


