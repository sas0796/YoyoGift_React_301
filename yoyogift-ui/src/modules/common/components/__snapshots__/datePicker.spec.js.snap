// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Date Picker Test Suite should render expected HTML 1`] = `"<form class=\\"DatePickers-container-1 [object Object]\\" novalidate=\\"\\"><div class=\\"MuiFormControl-root-3 DatePickers-textField-2 [object Object]\\"><label class=\\"MuiFormLabel-root-18 MuiInputLabel-root-7 MuiInputLabel-formControl-12 MuiInputLabel-animated-15 MuiInputLabel-shrink-14\\" data-shrink=\\"true\\" for=\\"date\\">undefined [object Object]</label><div class=\\"MuiInputBase-root-38 MuiInput-root-25 MuiInput-underline-29 MuiInputBase-formControl-39 MuiInput-formControl-26\\"><input aria-invalid=\\"false\\" class=\\"MuiInputBase-input-48 MuiInput-input-33 MuiInputBase-inputType-51 MuiInput-inputType-36\\" id=\\"date\\" type=\\"date\\" value=\\"undefined [object Object]\\"></div></div></form>"`;
